en/08_Changelogs/6.0.0.md

```diff
@@ -471,6 +472,17 @@ The one change we specifically want to call out is for [`ModelData::obj()`](api:
 
 See the [full list of removed and changed API](#api-removed-and-changed) to see all of the API with updated typing.
 
+### Changes to `LeftAndMain` and its subclasses {#leftandmain-refactor}
+
+[`LeftAndMain`](api:SilverStripe\Admin\LeftAndMain) has historically been the superclass for all controllers routed in `/admin/*` (i.e. all controllers used in the CMS). That class includes a lot of boilerplate functionality for setting up a menu, edit form, etc which a lot of controllers don't need.
+
+A new [`AdminController`](api:SilverStripe\Admin\AdminController) has been created which provides the `/admin/*` routing functionality and permission checks that `LeftAndMain` used to be responsible for. If you have a controller which needs to be routed through `/admin/*` with the relevant CMS permission checks, but which does *not* need a menu item on the left or an edit form, you should update that class to be a subclass of `AdminController` instead.
```

Comment:
Also not mentioning anything about formschema here because there's a follow-up card which will change where that code lives and I don't want to write a bunch of doc and then change it immediately afterward.